src/src/Helpers/StringConverter.cpp

```diff
@@ -790,6 +791,7 @@ void parseStandardConversions(String& s, boolean useURLencode) {
   SMART_CONV(F("%c_m2dhm%"),  minutesToDayHourMinute(arg1))
   SMART_CONV(F("%c_s2dhms%"), secondsToDayHourMinuteSecond(arg1))
   SMART_CONV(F("%c_2hex%"),   formatToHex(arg1, F("")))
+  SMART_CONV(F("%c_u2ip%"),   getIPAddressForUnit(arg1).toString())
```

Comment:
The `toString()` of IPAddress does not always output a string with ints and dots.
Maybe also check to see if it is set and output some expected value for unset values?

Reply:
I'll add a couple of options on what to return on invalid result